crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,14 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def stop_chef_service
+    ssh_cmd("/usr/sbin/rcchef-client stop")
```

Comment:
Maybe we can load chef client implementations here to invoke the correct method?